From: =?utf-8?q?David_Pr=C3=A9vot?= <taffit@debian.org>
Date: Fri, 25 Jul 2014 22:29:00 -0400
Subject: Document minimum-stability

Allow picky Composer parsers to actually rely on the currently
available 1.0.0beta1 version of PHP-Parser.

Forwarded: https://github.com/theseer/phpdox/pull/151
---
 composer.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/composer.json b/composer.json
index c21354c..32d4b0a 100644
--- a/composer.json
+++ b/composer.json
@@ -26,7 +26,7 @@
     "phpunit/php-timer" : ">=1.0.4",
     "zetacomponents/base": ">=1.8",
     "zetacomponents/console-tools" : ">=1.6.0",
-    "nikic/php-parser" : ">=1.0.0"
+    "nikic/php-parser" : ">=1.0.0@beta"
   },
   "autoload": {
     "classmap": [
